dataland-frontend/tests/component/components/resources/frameworkDataSearch/ThreeLayerDataTable.cy.ts

```diff
@@ -0,0 +1,147 @@
+import ThreeLayerDataTable from "@/components/resources/frameworkDataSearch/ThreeLayerDataTable.vue";
+import { minimalKeycloakMock } from "@ct/testUtils/Keycloak";
+import { euTaxonomyForNonFinancialsDisplayDataModel } from "@/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsDisplayDataModel";
+import { DataAndMetaInformationEuTaxonomyForNonFinancialsViewModel } from "@/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsViewModel";
+import { type DataAndMetaInformationEuTaxonomyDataForNonFinancials } from "@clients/backend";
+import { euTaxonomyForNonFinancialsModalColumnHeaders } from "@/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsModalColumnHeaders";
+
+describe("Component test for the NewEUTaxonomy Page", () => {
+  let mockedDataForTest: Array<DataAndMetaInformationEuTaxonomyForNonFinancialsViewModel>;
+
+  before(function () {
+    cy.fixture("EuTaxonomyForNonFinancialsMocks.json").then(
+      (mockedBackendResponses: DataAndMetaInformationEuTaxonomyDataForNonFinancials[]) => {
+        const singleMockDataAndMetaInfo = new DataAndMetaInformationEuTaxonomyForNonFinancialsViewModel(
+          mockedBackendResponses[0],
+        );
+        mockedDataForTest = [singleMockDataAndMetaInfo];
+      },
+    );
+  });
+
+  const expectedOrderOfCategories: string[] = ["BASIC INFORMATION", "ASSURANCE", "REVENUE", "CAPEX", "OPEX"];
+  const dataTestTagsOfCategories: string[] = ["Basic Information", "Assurance", "Revenue", "CapEx", "OpEx"];
```

Comment:
How about simply transforming all entries to all capital case and thereby use `dataTestTagsOfCategories` for `expectedOrderOfCategories`? Just thinking about minimizing the effort to add categories in case we want to at some point in time. However, the profit is very small so I would also be fine with how it is rn

Reply:
Due to the tight "deadline" I would suggest to just leave it like that, As said above, we wouldn't gain much from such a change